deployment/refinery-modules/refinery/manifests/init.pp

```diff
@@ -229,6 +226,7 @@
     Class["ui"],
     Class["solr"],
     Class["neo4j"],
+    Class["docker"],
```

Comment:
I'd remove that since there is no explicit dependency and docker installation will still happen by the time the Refinery manifest is applied.